jibri/rootfs/etc/cont-init.d/11-get-https-cert

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/with-contenv bash
+
+while ! openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null | openssl x509 -outform PEM > /usr/local/share/ca-certificates/meet.crt; do sleep 1; done
+update-ca-certificates
+# the screenshot is just to launch chrome and exit, to make it create ~/.pki
+su jibri -s /bin/sh -c '/usr/bin/google-chrome --headless --screenshot=/tmp/s.png https://meet.jitsi/ && certutil -d sql:/home/jibri/.pki/nssdb -A -t TC -n meet -i /usr/local/share/ca-certificates/meet.crt'
```

Comment:
Can you explain this sorcery? I love it!

Reply:
- while loop - retries obtaining the certificate in case the web container hasn't started yet.
- openssl inside while loop - the first openssl connects to web server and returns certificate on stdout, the second one saves the certificate in PEM format
- update-ca-certificates - redundant, can be removed. Chrome doesn't use system-wide certificate storage.
- su jibri -s /bin/sh -c - runs the quoted command as user jibri, to prevent permissions problems
- google-chrome --headless --screenshot ... - as explained in the comment
- certutil -d sql:/home/jibri/.pki/nssdb -A ... - adds certificate we've just obtained to the Chrome database